src/main/scala/loamstream/loam/intake/aggregator/AggregatorCommands.scala

```diff
@@ -0,0 +1,60 @@
+package loamstream.loam.intake.aggregator
+
+import loamstream.model.Store
+import loamstream.loam.LoamScriptContext
+import loamstream.loam.LoamSyntax
+import loamstream.loam.LoamCmdSyntax
+import loamstream.loam.intake.IntakeSyntax
+import java.nio.file.Paths
+
+/**
+ * @author clint
+ * Mar 2, 2020
+ */
+object AggregatorCommands extends AggregatorCommands
+
+trait AggregatorCommands {
+  import LoamSyntax._
+  import IntakeSyntax._
+  
+  def upload(
+      aggregatorIntakeConfig: AggregatorIntakeConfig,
+      metadata: Metadata, 
+      csvFile: Store, 
```

Comment:
Since it looks like this maps to the dig-aggregator-intake variants script file command, I just wanted to note that it's capable of `load`ing multiple CSV files and joining them by marker (`varId`). Maybe this is handled or isn't an issue here, but seeing only a single CSV file an input (or is this output?) I didn't want you painting yourself into a corner accidentally.

Reply:
I'll see what I can do here, and will certainly keep this in mind.  I've never run the Aggregator scripts in the mode you described.  In general, I tried to expose the minimal functionality required.  If it turns out that I need to `load` multiple CSV files at once, I'll make that possible.